nle/tests/test_db.py

```diff
@@ -26,7 +26,7 @@ def gen_ttyrecs(savedir):
         for _ in range(5):
             # Need to end naturally to be recorded.
             for c in [ord(" "), ord(" "), ord("<"), ord("y")]:
-                _, _, done, *_ = env.step(env.actions.index(c))
+                _, _, done, *_ = env.step(env.unwrapped.actions.index(c))
```

Comment:
Just out of curiosity, what makes the `unwrapped` necessary here and below?

Reply:
The Gymnasium API deprecated the original way of accessing wrapped attributes with the intention of removing it completely in the upcoming v1.0. Using `unwrapped` is their suggestion to properly access these.